virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,11 @@
+[id="virt-reclaiming-persistent-volumes"]
+= Reclaiming persistent volumes
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reclaiming-persistent-volumes
+toc::[]
+
+In order to reuse a statically-provisioned persistent volume (PV), you must first
+Reclaim the volume.
```

Comment:
Please confirm the styling on Reclaim. I think that it should be "reclaim."

Reply:
changed